crates/bifrost/src/providers/replicated_loglet/loglet.rs

```diff
@@ -121,6 +121,14 @@ impl<T: TransportConnect> ReplicatedLoglet<T> {
             log_server_manager,
         })
     }
+
+    pub(crate) fn is_sequencer_local(&self) -> bool {
+        matches!(self.sequencer, SequencerAccess::Local { .. })
+    }
+
+    pub(crate) fn known_global_tail(&self) -> TailOffsetWatch {
+        self.known_global_tail.clone()
```

Comment:
It doesn't look like we need a clone at all call-sites. Maybe just return a reference and the caller can decide to clone if they need to?